AdminWebsite/AdminWebsite/ClientApp/src/app/booking/hearing-schedule/date-error-messages.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { AbstractControl } from '@angular/forms';
+
+@Component({
+    selector: 'app-date-error-messages',
+    template: ` <div *ngIf="dateCtrl.errors?.required" class="alert alert-danger">
```

Comment:
could we move this into a seperate template file

Reply:
what extra value does it give??

Reply:
https://angular.io/guide/styleguide#extract-templates-and-styles-to-their-own-files